packages/fxa-event-broker/test/lib/selfUpdatingService/clientCapabilityService.spec.ts

```diff
@@ -53,21 +54,54 @@ describe('Client Capability Service', () => {
     });
   });
 
-  it('updates the service data', async () => {
-    service = new ClientCapabilityService(logger, {
+  it('fails on start if required to get capabilities', async () => {
+    nock('http://localhost')
+      .get('/v1/oauth/subscriptions/clients')
+      .reply(404, 'Not Found');
+    let error = null;
+    await service.start().catch(err => {
+      error = err;
+    });
+    cassert.isNotNull(error);
+    cassert.equal(((error as unknown) as Error).toString(), 'StatusCodeError: 404 - "Not Found"');
```

Comment:
What does `((error as unknown) as Error)` do?

Reply:
I couldn't cast it directly to Error since it was first infer'd as a null, so I had to cast to unknown first.